cs/src/json/JSON.csproj

```diff
@@ -19,16 +26,22 @@
     <Compile Include="protocols\SimpleJsonReader.cs" />
     <Compile Include="protocols\SimpleJsonWriter.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(BuildFramework)' == 'net45-nonportable'">
```

Comment:
Is it better to switch on `net45-nonportable` than something like `$(BuildNonportable) == true`?